app/controllers/admin/quickbooks_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Admin::QuickbooksController < Admin::AdminController
```

Comment:
Sorry I missed this but this should also be namespaced under `Admin::Accounting`.